parser/src/main/java/com/microsoft/gctoolkit/parser/ZGCParser.java

```diff
@@ -363,107 +381,150 @@ ZGCCycle toZGCCycle(DateTimeStamp endTime) {
         }
 
         public void setPauseMarkStart(DateTimeStamp pauseMarkStart) {
+            if (this.pauseMarkStart != null) {
+                return;
+            }
             this.pauseMarkStart = pauseMarkStart;
         }
 
         public void setPauseMarkStartDuration(double pauseMarkStartDuration) {
-            this.pauseMarkStartDuration = pauseMarkStartDuration;
+            this.pauseMarkStartDuration += pauseMarkStartDuration;
         }
 
         public void setPauseMarkEndStart(DateTimeStamp pauseMarkEndStart) {
+            if (this.pauseMarkEndStart != null) {
+                return;
+            }
             this.pauseMarkEndStart = pauseMarkEndStart;
         }
 
         public void setPauseMarkEndDuration(double pauseMarkEndDuration) {
-            this.pauseMarkEndDuration = pauseMarkEndDuration;
+            this.pauseMarkEndDuration += pauseMarkEndDuration;
```

Comment:
Is this GenZGC vs ZGC reasoning or fixing what you feel is a bug?

Reply:
Same with the other += changes.

Reply:
Due to genZ dividing pauseMarkEndDuration into young and old duration, so sum them equals the original meaning.
```
[4.120s][info][gc,phases   ] GC(0) Y: Pause Mark Start (Major) 0.066ms
......
[4.155s][info][gc,phases   ] GC(0) Y: Pause Mark End 0.016ms
......
[4.182s][info][gc,phases   ] GC(0) O: Pause Mark End 0.014ms
......
```